railties/test/application/configuration_test.rb

```diff
@@ -2593,6 +2593,20 @@ class MyLogger < ::Logger
       MESSAGE
     end
 
+    test "ActiveStorage.draw_routes can be configured via config.active_storage.draw_routes" do
+      app_file "config/environments/development.rb", <<-RUBY
+        Rails.application.configure do
+          config.active_storage.draw_routes = false
+        end
+      RUBY
+
+      output = rails("routes", "-g", "active_storage")
+      assert_equal <<~MESSAGE, output
+        No routes were found for this grep pattern.
```

Comment:
This test will break if we change the message. Can we make it less dependent on the message?